fly/commands/sync.go

```diff
@@ -10,14 +10,18 @@ import (
 	"github.com/vbauerster/mpb/v4/decor"
 
 	"github.com/concourse/concourse"
+	"github.com/concourse/concourse/atc"
 	"github.com/concourse/concourse/fly/commands/internal/displayhelpers"
 	"github.com/concourse/concourse/fly/rc"
 	"github.com/concourse/concourse/fly/ui"
 )
 
 type SyncCommand struct {
-	Force  bool   `long:"force" short:"f" description:"Sync even if versions already match."`
-	ATCURL string `long:"concourse-url" short:"c" description:"Concourse URL to sync with"`
+	Force          bool         `long:"force" short:"f" description:"Sync even if versions already match."`
+	ATCURL         string       `long:"concourse-url" short:"c" description:"Concourse URL to sync with"`
+	CACert         atc.PathFlag `long:"ca-cert" description:"Path to Concourse PEM-encoded CA certificate file."`
```

Comment:
Good catch. Maybe it's also worth adding `Insecure` for consistency with `fly login`?

Reply:
Oh, yeah. I forgot to mention that I did a drive-by on that too.

Yeah, I'll put insecure in too on the next push.